docs/guides.rst

```diff
@@ -5,6 +5,54 @@ Guides
 
 These guides will help you configure Chassis for specific types of development.
 
+Migrations
+----------
+
+Importing A Production Database Into Chassis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Provision a new Chassis instance.
+#. Export your production database with `WP-CLI`_ ``wp db export <filename.sql>``.
```

Comment:
I've had problems with multi-byte characters using this command but can't remember the export command I used.

Cool story.

Reply:
Hmm. My guess is it was probably utf8mb4 going into utf8. It's an edge case so if someone hits it hopefully they'll open an issue and I can update the docs.